### PR TITLE
refactor: Remove private methods from the client to separate modules

### PR DESCRIPTION
**Pull request by dtroelofsprins**

### CODE REVIEW

File: molgenis/utils.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+            print(attribute)
+            one_to_manys.append(attribute)
+    copied_rows = copy.deepcopy(rows)
+    for row in copied_rows:
+        for one_to_many in one_to_manys:
+            row.pop(one_to_many, None)
+    return copied_rows
+
+
+def set_urls(url: str):
```
Comment (tommydeboer) [inline molgenis/utils.py:119]: This method is so specific to the Session class that I think it should be kept there. Nobody else will probably ever have use for this.

File: molgenis/client.py [hunk 2]
```diff
@@ -64,18 +37,12 @@ def __init__(self, url: str = "http://localhost:8080/", token: str = None):
         Examples:
         >>> session = Session('http://localhost:8080/')
         """
-        self._set_urls(url)
+        self._api_url = utils.set_urls(url)[0]
+        self._root_url = utils.set_urls(url)[1]
```
Comment (tommydeboer) [inline molgenis/client.py:41]: You're calculating the urls twice here. You need to store the result of `set_urls` in a temporary variable, or you can unpack the returned tuple like this:

```python
self._api_url, self._root_url = utils.set_urls(url)
```

File: molgenis/get_utils.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
+from typing import List, Optional
```
Comment (tommydeboer) [inline molgenis/get_utils.py:1]: Shall we rename to `query_utils.py`? It's a little more descriptive

**Review by tommydeboer — CHANGES_REQUESTED** [review]

File: molgenis/utils.py (hunk 1 above)
Comment (dtroelofsprins) [reply molgenis/utils.py]: I was indeed also doubting about this one, tried several ways of including it in a different module, finally did it like this, but found the current way actually still better.

File: molgenis/client.py (hunk 2 above)
Comment (dtroelofsprins) [reply molgenis/client.py]: Won't fix because the current way of defining these urls is restored.

File: molgenis/get_utils.py (hunk 3 above)
Comment (dtroelofsprins) [reply molgenis/get_utils.py]: Good suggestion!

**Review by tommydeboer — APPROVED** [review]

**Pull request merged**